src/tidy.zig

```diff
@@ -367,7 +367,7 @@ fn tidy_dead_declarations(
 }
 
 /// As we trim our functions, make sure to update this constant; tidy will error if you do not.
-const function_line_count_max = 459; // fn check in state_machine.zig
+const function_line_count_max = 413; // fn check in state_machine.zig
```

Comment:
Nice win! We should get back to our function length at some point, but not today!